### DISCUSSION TITLE
How to get Customer payment receipt

### DISCUSSION THREAD

**Discussion by zycabathware** [Erratums/ims-support#76] [Q&A] [answered]
We cannot find any button to print a receipt for customer payments. 
How can we print or share a PDF with customer

**Comment by erratumsolutions** [comment]
We currently do not have this option. We will be adding this soon

**Comment by erratumsolutions** [comment, marked as answer]
A task has been created for this [here](https://github.com/Erratums/ims-support/issues/78)
